Bonobo.Git.Server/Git/GitLfs/LfsAppDataStorageProvider.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Web;
+
+namespace Bonobo.Git.Server.Git.GitLfs
+{
+    /// <summary>
+    /// Provides a simple LfsDataStorageProvider implementation where LFS objects are stored directly in the AppData folder of the Bonobo web site.
+    /// </summary>
+    public class LfsAppDataStorageProvider : ILfsDataStorageProvider
+    {
+        public string GetFileUrl(string urlScheme, string urlAuthority, string requestApplicationPath, string operation, string repositoryName, string oid, long size)
+        {
+            string url = string.Concat(
```

Comment:
The UriBuilder class is usually what you want to build your URL.

Reply:
This is now using a UriBuilder except for the path construction.